Hurd: comment PF_ROUTE/AF_ROUTE defines

Comment out the PF_ROUTE and AF_ROUTE defines, since they would be used for PF_ROUTE setsockopts, which are not available on Hurd.

Index: glibc-2.37/sysdeps/mach/hurd/bits/socket.h
===================================================================
--- glibc-2.37.orig/sysdeps/mach/hurd/bits/socket.h
+++ glibc-2.37/sysdeps/mach/hurd/bits/socket.h
@@ -95,7 +95,7 @@ enum __socket_type
 #define	PF_LAT		14	/* DEC Local Area Transport protocol.  */
 #define	PF_HYLINK	15	/* NSC Hyperchannel protocol.  */
 #define	PF_APPLETALK	16	/* Don't use this.  */
-#define	PF_ROUTE	17	/* Internal Routing Protocol.  */
+/* #define	PF_ROUTE	17	Internal Routing Protocol.  */
 /* #define	PF_LINK		18	Link layer interface.  */
 #define	PF_XTP		19	/* eXpress Transfer Protocol (no AF).  */
 #define	PF_COIP		20	/* Connection-oriented IP, aka ST II.  */
@@ -128,7 +128,7 @@ enum __socket_type
 #define	AF_LAT		PF_LAT
 #define	AF_HYLINK	PF_HYLINK
 #define	AF_APPLETALK	PF_APPLETALK
-#define	AF_ROUTE	PF_ROUTE
+/* #define	AF_ROUTE	PF_ROUTE */
 /* #define	AF_LINK		PF_LINK */
 #ifdef __USE_MISC
 # define	pseudo_AF_XTP	PF_XTP
